template/.gitignore

```diff
@@ -105,3 +105,5 @@ vendor/.git
 nbproject
 nbproject/*
 
+# DrupalVM
+.vagrant/
```

Comment:
Drupal VM has two user specific configs as well: `local.config.yml` and `Vagrantfile.local`.